pyrene/src/components/Collapsible/Collapsible.tsx

```diff
@@ -66,19 +66,19 @@ const Collapsible: React.FC<CollapsibleProps> = ({
   };
 
   return (
-    <div styleName={clsx('collapsibleBox', { expanded })}>
-      <div styleName={clsx('buttonAlignmentBox', { [`align-${align}`]: true })}>
-        <div styleName="collapsibleButton" className="unSelectable" onClick={(event:React.MouseEvent) => toggleCollapse(event)} role="button" aria-label="Show or hide content">
-          <div styleName="centeringBox">
-            <span styleName="label">
+    <div className={clsx(styles.collapsibleBox, { [styles.expanded]: expanded })}>
+      <div className={clsx(styles.buttonAlignmentBox, { [styles[`align-${align}`]]: true })}>
```

Comment:
Please do the following when the `class` will be added anyway, means always `true`.

```
<div className={clsx(styles.buttonAlignmentBox, styles[`align-${align}`] )}>
```